vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -191,7 +191,9 @@ $global:map = @{
         "https://acs-mirror.azureedge.net/kubernetes/v1.24.0/windowszip/v1.24.0-1int.zip"
     );
     "c:\akse-cache\win-vnet-cni\" = @(
-        "https://acs-mirror.azureedge.net/azure-cni/v1.4.22/binaries/azure-vnet-cni-singletenancy-windows-amd64-v1.4.22.zip"
+        "https://acs-mirror.azureedge.net/azure-cni/v1.4.22/binaries/azure-vnet-cni-singletenancy-windows-amd64-v1.4.22.zip",
+        "https://acs-mirror.azureedge.net/azure-cni/v1.4.29/binaries/azure-vnet-cni-singletenancy-windows-amd64-v1.4.29.zip",
```

Comment:
it does not exist in sov cloud.
https://mirror.azk8s.cn/azure-cni/azure-cni/v1.4.29/binaries/azure-vnet-cni-singletenancy-windows-amd64-v1.4.29.zip

Reply:
@jaer-tsun how should we use swift cni in Windows nodes? do you plan to update Windows cse scripts?

Reply:
there's a typo in the url you pasted (extra /azure-cni), I'm able to wget

Reply:
we'll be using it for [windows AKS-Swift](https://msazure.visualstudio.com/DefaultCollection/CloudNativeCompute/_git/aks-rp/pullrequest/5914847)

Reply:
> https://mirror.azk8s.cn/azure-cni/azure-cni/v1.4.29/binaries/azure-vnet-cni-singletenancy-windows-amd64-v1.4.29.zip

My mistake. https://mirror.azk8s.cn/azure-cni/v1.4.29/binaries/azure-vnet-cni-singletenancy-windows-amd64-v1.4.29.zip works.

Reply:
> we'll be using it for [windows AKS-Swift](https://msazure.visualstudio.com/DefaultCollection/CloudNativeCompute/_git/aks-rp/pullrequest/5914847)

Will discuss this in the RP PR.